hooks/playbooks/ceph-bm.yml

```diff
@@ -0,0 +1,39 @@
+- name: Deploy ceph on OSP BMO deployed nodes
+  hosts: "{{ cifmw_target_hook_host | default('localhost') }}"
+  gather_facts: false
+  tasks:
+    - name: Fetch OSP BMO nodesets
+      environment:
+        KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
+        PATH: "{{ cifmw_path }}"
+      ansible.builtin.command:
+        cmd: >-
+          oc get OpenStackBaremetalSet -n "{{ _namespace}}"  -o yaml
```

Comment:
I see this is one command to generate a yaml file so it's probably fine.

Did you consider using the `kubernetes.core.k8s` module? For example.

https://github.com/openstack-k8s-operators/ci-framework/blob/main/roles/ci_lvms_storage/tasks/main.yml#L42-L52'

In general I think this is nicer than running oc commands but I don't think we have to block the review over it.

Reply:
I'll look into it but I would prefer doing it after getting this patch merge having that those are leftovers we wanted to get done ASAP.